bundles/mapping/mapmodule/plugin/vectortilelayer/VectorTileModelBuilder.js

```diff
@@ -2,15 +2,10 @@ import { VectorStyle } from '../../domain/VectorStyle';
 export class VectorTileModelBuilder {
     parseLayerData (layer, mapLayerJson, maplayerService) {
         const { options = {}, style } = mapLayerJson;
-        const { styles = {}, externalStyles = {}, hover } = options;
-
-        Object.keys(styles).forEach(name => {
-            const style = new VectorStyle(name, null, 'normal', styles[name]);
-            layer.addStyle(style);
-        });
-        // Remove styles from options to be sure that VectorStyle is used
-        delete options.styles;
+        const { externalStyles = {}, hover } = options;
 
+        // normal styles are set in AbstractVectorLayer setOptions
+        // set external styles
         Object.keys(externalStyles).forEach(name => {
```

Comment:
We could do the same thing for these external styles: override setOptions() for VectorTileLayer and call super.setOptions() to get the non-externalstyles parsed in the layer (instead of modelbuilder) so we don't run in the same problem later when vectortilelayer styles are updated by the admin/user?